addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/handler/PanelThingHandler.java

```diff
@@ -180,76 +182,70 @@ public void updateChannel(ChannelUID channelUID, int state, String description)
      */
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        if (dscAlarmBridgeHandler == null) {
-            logger.warn("DSC Alarm bridge handler not available. Cannot handle command without bridge.");
-            return;
-        }
 
-        int cmd;
+        logger.debug("handleCommand(): Command Received - {} {}.", channelUID, command);
 
-        boolean connected = dscAlarmBridgeHandler.isConnected();
+        if (command instanceof RefreshType) {
+            return;
+        }
 
-        if (connected) {
-            switch (channelUID.getId()) {
-                case PANEL_COMMAND:
-                    cmd = Integer.parseInt(command.toString());
-                    switch (cmd) {
-                        case 0:
-                            dscAlarmBridgeHandler.sendCommand(DSCAlarmCode.Poll);
-                            break;
-                        case 1:
-                            dscAlarmBridgeHandler.sendCommand(DSCAlarmCode.StatusReport);
-                            break;
-                        case 2:
-                            dscAlarmBridgeHandler.sendCommand(DSCAlarmCode.LabelsRequest);
-                            break;
-                        case 8:
-                            dscAlarmBridgeHandler.sendCommand(DSCAlarmCode.DumpZoneTimers);
-                            break;
-                        case 10:
-                            dscAlarmBridgeHandler.sendCommand(DSCAlarmCode.SetTimeDate);
-                            break;
-                        case 200:
-                            dscAlarmBridgeHandler.sendCommand(DSCAlarmCode.CodeSend, getUserCode());
-                            break;
-                        default:
-                            break;
-                    }
+        if (dscAlarmBridgeHandler != null) {
+
+            if (dscAlarmBridgeHandler.isConnected()) {
+                int cmd;
+
+                switch (channelUID.getId()) {
+                    case PANEL_COMMAND:
+                        cmd = Integer.parseInt(command.toString());
+                        switch (cmd) {
+                            case 0:
```

Comment:
To me this values look quite random it's a common best practice to create named constants in such cases.